tests/test_middleware_interface.py

```diff
@@ -1001,6 +1003,29 @@ def _make_expanded_ref(registry, dtype, data_id, run):
         return lsst.daf.butler.DatasetRef(registry.getDatasetType(dtype), data_id, run=run) \
             .expanded(registry.expandDataId(data_id))
 
+    @staticmethod
+    def _make_test_graph(n_quanta):
+        """Make a `QuantumGraph` for tests.
+
+        Parameters
+        ----------
+        n_quanta : `int`
+            Number of quanta in a graph.
+
+        Returns
+        -------
+        qgraph : `~lsst.pipe.base.QuantumGraph`
+            Quantum graph instance.
+        """
+        with tempfile.TemporaryDirectory() as tmpdir:
+            if n_quanta == 0:
+                butler = Butler(Butler.makeRepo(tmpdir))
+                return lsst.pipe.base.QuantumGraph({}, universe=butler.dimensions)
```

Comment:
It seems a bit gratuitous to make a temporary repo just to get a QuantumGraph that doesn't use it. Can `DimensionUniverse` not be constructed directly?

Reply:
Yes. If you don't care about a specific universe then `universe=DimensionUniverse()` is all you need here.

Reply:
Thank you. That's definitely better. 

Modified.